### ISSUE TITLE
Final Review Request of seven (7) W3C VCWG Specifications

### ISSUE THREAD

**Issue by msporny** [w3c/a11y-request#99]
The W3C VCWG is requesting a *final* horizontal review of the following specifications, which your group has reviewed previous versions of over the past year or more.

We are requesting this before proceeding to the Proposed Recommendation phase based on a request by W3M to ensure that changes that have been made in the past six months are reviewed by your group since your last review. We are planning to enter the Proposed Recommendation phase for all seven (7) documents in Q1 2025 (roughly February, if possible).

 - [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-data-model-2.0/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-data-model-2.0-test-suite/)
 - [Verifiable Credential Data Integrity v1.0](https://www.w3.org/TR/vc-data-integrity/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-data-integrity/#revision-history)
   - Implementation Report (see ECDSA and EdDSA test suites, which test the Data Integrity base spec in each cryptosuite)
 - [ECDSA Cryptosuite v1.0](https://www.w3.org/TR/vc-di-ecdsa/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-di-ecdsa/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-di-ecdsa-test-suite/)
 - [EdDSA Cryptosuite v1.0](https://www.w3.org/TR/vc-di-eddsa/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-di-eddsa/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-di-eddsa-test-suite/)
 - [Securing Verifiable Credentials using JOSE and COSE](https://www.w3.org/TR/vc-jose-cose/)
   - [Changes since Candidate Recommendation](https://github.com/w3c/vc-jose-cose/commits/?since=2024-09-15&until=now)
   - [Implementation Report](https://w3c.github.io/vc-jose-cose-test-suite/)
 - [Bitstring Status List v1.0](https://www.w3.org/TR/vc-bitstring-status-list/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/vc-bitstring-status-list/#revision-history)
   - [Implementation Report](https://w3c.github.io/vc-bitstring-status-list-test-suite/)
 - [Controlled Identifier Documents v1.0](https://www.w3.org/TR/controller-document/)
   - [Changes since Candidate Recommendation](https://www.w3.org/TR/controller-document/#revision-history)
   - [Implementation Report](https://w3c.github.io/cid/implementations/1.0/)

  - Primary contacts:
      - Brent Zundel @brentzundel (Chair)
      - Ivan Herman @iherman (W3C Staff Contact)
      - Greg Bernstein @wind4greg (Editor, Data Integrity and cryptosuite specs)
      - Mike Jones @selfissued (Editor, VCDM v2.0 and VC JOSE COSE spec)
      - Gabe Cohen @decentralgabe (Editor, VC JOSE COSE spec)
      - Manu Sporny @msporny (Editor, VCDM v2.0, Data Integrity, Bitstring Status List specs)
      - Dmitri Zagidulin @dmitrizagidulin (Editor, Data Integrity and cryptosuite specs)
  - Organization/project driving the specification:
      - W3C Verifiable Credentials Working Group

Further details:

There are no major unresolved issues or opposition to these specifications that we know of at the present time.

**Comment by lwolberg** [comment]
On behalf of APA, we have reviewed these 7 documents and are pleased to recommend all of them for advancement. 

We do request, if readily achievable, that we loosen the statements regarding WCAG. To be specific, two of the documents refer and link to WCAG 2.1, which is now superseded by WCAG 2.2. We have two proposed options for rectifying this, if readily achievable. 


* [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/)
Accessibility Considerations current text: It is important to follow accessibility guidelines and standards, such as WCAG22. 
OPTION 1: Change it to WCAG 2.2. URL is, https://www.w3.org/TR/WCAG22/
OPTION 2: Change to broader language pointing to the latest version of WCAG. APA Chairs, please supply suitable text. 

* [Securing Verifiable Credentials using JOSE and COSE](https://www.w3.org/TR/vc-jose-cose/)
Accessibility Considerations current text: it is vital to adhere to accessibility guidelines and standards, such as the Web Content Accessibility Guidelines (WCAG 2.1).
OPTION 1: Change it to WCAG 2.2. URL is, https://www.w3.org/TR/WCAG22/
OPTION 2: Change to broader language pointing to the latest version of WCAG. APA Chairs, please supply suitable text. 


We repeat: We are happy with the documents' accessibility considerations sections, and request these changes only if the Working Group finds it within their ability to make.

**Comment by msporny** [comment]
Wonderful! Thank you @lwolberg and APA for the timely review of all documents. 

We can do either Option 1 or option 2 and would prefer to change to broader language that points to the latest version of WCAG in order to make the language/guidance more future proof (that is, to avoid the current oversight).

**Comment by JaninaSajka** [comment]
Here's the appropriate link for Option 2: [https://www.w3.org/TR/WCAG/](current WCAG). Very glad to hear this edit makes sense to everyone!

**Comment by msporny** [comment]
I'll make sure this change happens one way or the other. 

Here's the problem: Because we use ReSpec for our spec, it uses specref.org to look up spec references. The current references for WCAG is an alias for WAI-WEBCONTENT 

https://www.specref.org/?q=WCAG

which points to:

```Wendy Chisholm; Gregg Vanderheiden; Ian Jacobs. Web Content Accessibility Guidelines 1.0. 5 May 1999. W3C Proposed Recommendation. URL: https://www.w3.org/TR/WAI-WEBCONTENT/```

... which is from the year 1999, not good.

We wanted to use the latest, which at the time was WCAG21. This is problematic because of the issue raised here. We have to update the reference by hand every time WCAG changes and that is an error prone human process.

Now, this is just one reference, so I could just use the URL you mention above and direct link to it, but this won't fix the problem for other WGs and I expect this to keep resurfacing again.

I expect the right thing to do is to update the [WCAG] alias to point to https://www.w3.org/TR/WCAG/ -- I don't know how to do that in specref.org. 

In any case, I'll fix the link in the VCWG specs, but that won't fix the underlying problem and I expect other WGs to be affected by this particular issue.

**Comment by msporny** [comment]
> This link, without the version tailend, will always point to latest WCAG

Yes, I realize that -- however, when people use specref.org or ReSpec to cite WCAG, like so in the HTML source: [WCAG] or [[[WCAG]]] - it will point to the wrong location.

**Comment by iherman** [comment]
I would think a pull request should be made on https://github.com/tobie/specref/blob/main/refs/biblio.json#L3560 and change whatever is necessary to be changed.

**Comment by ruoxiran** [comment]
APA is good with those documents, so please go ahead. thank you.
